packages/nixd/package.yaml

```diff
@@ -0,0 +1,26 @@
+---
+name: nixd
+description: Language Server for Nix.
+homepage: https://github.com/nix-community/nixd
+licenses:
+  - LGPL-3.0
+languages:
+  - Nix
+categories:
+  - LSP
+
+source:
+  # renovate:datasource=github-tags
+  id: pkg:github/nix-community/nixd@1.2.3
```

Comment:
It this version outdated?

Reply:
At the time of this MR it wasn't.

But this should now be 2.1.2

Reply:
I've opened a new PR at 2.4.0. Happy to close if this PR ends up being updated.

Cheers.